src/Cache.php

```diff
@@ -4,29 +4,39 @@
 
 namespace Interns2022B;
 
+use Illuminate\Support\Collection;
+use Interns2022B\Models\Brewery;
+
 class Cache
 {
     public const CLEAR = "clear";
     public const BUILD = "build";
     public const TABLE_DIRECTORY = __DIR__ . "/../tests/stubs/table.json";
 
-    public array $toFile = [];
+    private array $toFile = [];
 
-    public function putData(string $provider, array $breweriesData): void
+    public function putData(string $provider, Collection $breweriesFactory): void
     {
-        foreach ($breweriesData as $rowValue) {
-            if ($rowValue["provider"] === $provider) {
-                $this->toFile[] = $rowValue;
+        $breweriesFactory->each(function (Brewery $item) use ($provider): void {
+            if ($item->providers->first() === $provider) {
+                $add = [
+                    "name" => $item->name,
+                    "city" => $item->city->name,
+                    "country" => $item->countries->first(),
+                    "provider" => $item->providers->first(),
+                ];
+                $this->toFile[] = $add;
             }
-        }
+        });
     }
 
-    public function rebuildCache(array $providers, array $breweriesData): void
+    public function rebuildCache(Collection $providers, Collection $breweriesFactory): void
```

Comment:
I would like to rename breweriesFactory to breweries
Let's add doc above method name: 
```
/** 
* @var Collection<Provider> $providers 
* @var Collection<Brewery> $breweries 
* /
```

Reply:
Renamed but composer deletes the doc